CONTRIBUTING.md

```diff
@@ -410,66 +410,79 @@ If you are adding a new service directory, ensure that it is mapped to a friendl
 
 ## On-boarding New generated code library
 
-1. Make a copy of `/sdk/template/Azure.Template` in you appropriate service directory and rename projects to `Azure.Management.*` for management libraries or `Azure.*` (e.g.  `sdk/storage/Azure.Management.Storage` or `sdk/storage/Azure.Storage.Blobs`)
+1. Make a copy of `sdk/template/Azure.Template` in you appropriate service directory and rename projects to `Azure.Management.*` for management libraries or `Azure.*` (e.g.  `sdk/storage/Azure.Management.Storage` or `sdk/storage/Azure.Storage.Blobs`).
+
+   Alternatively, you can install `sdk/template` as a `dotnet new` template to create or update a service directory:
```

Comment:
For mgmt isn't there a different template at https://github.com/Azure/azure-sdk-for-net/tree/main/eng/templates? I've not looked close but is there any good way to share them or are they 2 different to parameterize?

Reply:
There is. I can list both here as well.